src/reducers/cars.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+	cars: [{
+		brand: "VW",
+		model: "Passat",
+	},
+	{
+		brand: "Audi",
+		model: "A4",
+	}]
+}
+export default function carsList(state = initialState, action){
+	switch (action.type) {
+	case "SET_CARS":
+		return { ...state, cars: action.payload }
```

Comment:
does it work correctly?
i think you can 
`return ...action.payload`
but maybe i'm wrong, so just check it